Budget/DayBudget/DayBudget.swift

```diff
@@ -9,6 +9,8 @@
 import Foundation
 
 public class DayBudget {
```

Comment:
You mean a struct? Yeah there is, expenses should be added dring the lifetime of the dayBudget. So it makes more sense to just have one DayBudget that changes instead of copying it

Reply:
A struct can change as well. You're proving my point here. ;)

Reply:
I can't add expenses to the dayBudget when its a struct, then i'll have to make the dayBudget a var.

What do you mean im proving your point?